Modules/Loadable/Markups/Widgets/Resources/UI/qSlicerSimpleMeasurementsWidget.ui

```diff
@@ -0,0 +1,598 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>qSlicerSimpleMeasurementsWidget</class>
+ <widget class="qSlicerWidget" name="qSlicerSimpleMeasurementsWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>300</width>
+    <height>392</height>
+   </rect>
+  </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>300</width>
+    <height>0</height>
+   </size>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <item>
+    <layout class="QVBoxLayout" name="widgetVerticalLayout">
+     <item>
+      <widget class="ctkCollapsibleGroupBox" name="createMarkupsGroupBox">
+       <property name="styleSheet">
+        <string notr="true">QGroupBox {
+        border: 2px solid rgb(127,127,17);
+        border-radius: 3px;
+    }
+QGroupBox::title {
+        subcontrol-origin: margin;
+        subcontrol-position: top left;
+    }</string>
+       </property>
+       <property name="title">
+        <string>Add</string>
```

Comment:
“Create” is used in the markups module. “Add” can imply something already exists so is not as good of a term as “Create” here.
```suggestion
        <string>Create</string>
```

Reply:
I have applied the correction. Thanks for the feedback!

Reply:
I have applied the correction. Thanks for the feedback!